externs/shaka/player.js

```diff
@@ -815,6 +816,10 @@ shaka.extern.DrmConfiguration;
  *   Called immediately after the DASH manifest has been parsed into an
  *   XMLDocument. Provides a way for applications to perform efficient
  *   preprocessing of the manifest.
+ * @property {boolean} sequenceMode
+ *   If true, the media segments are appended to the SourceBuffer in strict
+ *   sequence. Defaults to <code>true</code> except on WebOS 3, Tizen 2,
```

Comment:
Why does this default to `true`?  I don't think we should do that for DASH.

DASH is designed to communicate timestamps in the manifest.  HLS is built off of a playlist format, to play segments in sequence.  I think sequence mode as a default makes much more sense for HLS than DASH.